src/main/java/uk/gov/companieshouse/reconciliation/config/ComparisonGroupModel.java

```diff
@@ -20,10 +30,18 @@ public void setGroupName(String groupName) {
         this.groupName = groupName;
     }
 
+    /**
+     * @return size representing the number email link models in a comparison group
+     */
     public int getSize() {
         return emailLinkModel.size();
     }
 
+    /**
+     * Returns Map of {@link EmailLinkModel}'s indexed by emailId.
```

Comment:
Please remove the apostrophe from EmailLinkModel's.

Reply:
Resolved in latest commit.